pkg/handlers/supportapi/payment_request.go

```
@@ -399,21 +399,21 @@ func (h ProcessReviewedPaymentRequestsHandler) Handle(params paymentrequestop.Pr
 
 				sshClient, err := cli.InitGEXSSH(appCtx, v)
 				if err != nil {
-					appCtx.Logger().Fatal("couldn't initialize SSH client", zap.Error(err))
+					appCtx.Logger().Error("couldn't initialize SSH client", zap.Error(err))
```

Don't you need to return the error here and in the other similar situations below?